contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/ParentNode.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+
+  <VListGroup
+    v-model="open"
+    append-icon="arrow_drop_down"
+    class="parent-item"
+    lazy
+    sub-group
+  >
+    <template v-slot:activator>
+      <ContentNode
+        :nodeId="nodeId"
+        :sourceId="sourceId"
+      >
+        <VListTileContent class="description-col py-2 pl-2 shrink">
+          <VBadge color="primary">
+            <template #badge>
+              <span class="caption font-weight-bold">{{ descendantCount }}</span>
+            </template>
+            <VListTileTitle class="text-truncate">
+              {{ contentNode.title }}
+            </VListTileTitle>
+          </VBadge>
+        </VListTileContent>
+
+        <!-- Custom placement of dropdown indicator -->
+        <VListTileAction
+          class="v-list__group__header__append-icon action-col px-1"
+        >
+          <Icon>arrow_drop_down</Icon>
+        </VListTileAction>
+        <VSpacer />
+      </ContentNode>
+    </template>
+
+    <transition-group>
+      <template v-for="child in treeChildren">
+        <ParentNode
+          v-if="hasChildren(child.id)"
+          :key="child.id"
+          :nodeId="child.id"
+          :sourceId="child.source_id"
+        />
+        <ContentNode
+          v-else
+          :key="child.id"
+          :nodeId="child.id"
+          :sourceId="child.source_id"
+        />
+      </template>
+    </transition-group>
+
+  </VListGroup>
+
+</template>
+<script>
+
+  import ContentNode from './ContentNode';
+  import clipboardMixin, { parentMixin } from './mixins';
+
+  export default {
+    name: 'ParentNode',
```

Comment:
Sorry, this feels super nitpicky to me, but could you rename this to TopicNode so it's more consistent with the way we reference it throughout the platform?

Reply:
No problem, I had anticipated reusing this for the channel, but that didn't happen.